Shared/Tests.Sync.Shared/MergeByPKTests.cs

```diff
@@ -30,6 +30,16 @@ namespace Tests.Sync
     [TestFixture, Preserve(AllMembers = true)]
     public class MergeByPKTests
     {
+        [SetUp]
+        public void SetUp()
+        {
+            // We simulate a clean slate first time user login
```

Comment:
Maybe this means we should move this code to `ResetForTesting`? It's what realm-js does.